### PR TITLE
Move register

### PR DESCRIPTION
**Pull request by anowakowski**
create new register view with material design. Its new component like login. Prepare validation and other smaller functionality for login and register pages

### CODE REVIEW

File: src/app/modules/register-login/components/register/register.component.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+import { User } from 'src/app/models/user';
+import { AuthService } from 'src/app/services/auth.service';
+import { Router } from '@angular/router';
+import { SnacbarAlertService } from 'src/app/services/snacbar-alert.service';
+import { NgxSpinnerService } from 'ngx-spinner';
+
+const REQUIRES_MESSAGE = 'You must enter a value';
+const NOTVALID_EMAIL_MESSAGE = 'Not a valid email';
+const BLANK_MESSAGE = '';
+const NOTVALID_PASSWORDMATCHES_MESSAGE = 'Password dont matches';
```
Comment (mvein) [inline src/app/modules/register-login/components/register/register.component.ts]: Typo: lack of apostrophe in word don't

File: src/app/modules/home/components/home-section/homes-section.component.html [hunk 2]
```diff
@@ -34,15 +34,11 @@ <h1 class="display-4">Get
...
-          <div class="card register-signup text-center card-form">
-            <div class="card-body">
-              <h3>Sign Up Today</h3>
-              <p>Please fill out this form to register</p>
-              <app-register-form></app-register-form>
-            </div>
-          </div>
+        <div class="col text-center py-5">
+          <h1 class="display-4">SignUp</h1>
+          <p class="lead">Go to regiseter page and sign up, or login if you are existing user</p>
```
Comment (mvein) [inline src/app/modules/home/components/home-section/homes-section.component.html]: Typo: should be register instead of 'regiseter'.

Comment (anowakowski) [reply src/app/modules/home/components/home-section/homes-section.component.html]: fixed by:
https://github.com/anowakowski/PictureApp_SPA/pull/25/commits/c756011b6cfe7fe12c9a6b15ddaa543cb1ec0117

File: src/app/modules/register-login/components/register/register.component.ts (hunk 1 above)
Comment (anowakowski) [reply src/app/modules/register-login/components/register/register.component.ts]: fixed by:
https://github.com/anowakowski/PictureApp_SPA/pull/25/commits/c756011b6cfe7fe12c9a6b15ddaa543cb1ec0117

**Pull request merged**